packages/vscode-graphql-execution/package.json

```diff
@@ -110,6 +110,7 @@
     "graphql-tag": "2.12.6",
     "graphql-ws": "5.10.0",
     "@whatwg-node/fetch": "0.2.8",
+    "wonka": "4.0.15",
```

Comment:
added in dependency since `wonka` is imported here https://github.com/graphql/graphiql/blob/6c3ca902d769893a4201f8856299af18b7e21a1b/packages/vscode-graphql-execution/src/helpers/network.ts#L6

Reply:
Esbuild seems to bundle the `@urql` provided version. I'm wary of managing this version in case it conflicts with `@urql` at some point though we are pining. 
Perhaps we can import these methods directly from some `@urql` package now?

Reply:
after removing `wonka` and adding `eslint-disable` comment, deploy passes 🤔

Reply:
I removed wonka from dependencies, I guess after switching to pnpm we will still need to add it to the dependencies